### ISSUE TITLE
Narrator doesn't announce the suggestions if it matches with any entry in the Filter By...textboxes in ListView.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/WinUI-Gallery#334]
**Describe the bug**

**Steps to reproduce the bug**

Prerequisite: 
* Turn on Narrator.
* Keep narrator in item mode.

Repro steps :
1. Launch XAML Controls Gallery.
2. Activate ListView from home page.
3. Navigate to "Filter By" textboxes under the header "ListView with filtering" and enter any data in the textboxes and observe the narrator's description.

**Expected behavior**

Narrator should announce the suggestions if it matches with any entry in the Filter By...textboxes.

**Actual behavior**

Narrator does not announce the suggestions available.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/73033903-50807100-3df8-11ea-8f05-160145b29e69.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/24662739)
